### PR TITLE
source-illumina-basespace contribution from FilipeJesus

### PR DESCRIPTION
**Pull request by FilipeJesus**
Auto-generated PR by the Connector Builder for source-illumina-basespace

### CODE REVIEW

**Comment by erohmensing** [comment]
@FilipeJesus cool connector, thanks for the contribution! Do you have an icon SVG you'd like the connector to use that you could share?

**Comment by FilipeJesus** [comment]
> @FilipeJesus cool connector, thanks for the contribution! Do you have an icon SVG you'd like the connector to use that you could share?

Hi @erohmensing, I have updated the SVG icon to the illumina logo.

File: airbyte-integrations/connectors/source-illumina-basespace/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+metadataSpecVersion: "1.0"
+data:
+  allowedHosts:
+    hosts:
+      - "*"
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-illumina-basespace/metadata.yaml:5]:
```suggestion
      - "api.euw2.sh.basespace.illumina.com"
```

**Comment by natikgadzhi** [comment]
@FilipeJesus thank you for this! <3

**Pull request merged**